Pod/ILGClasses/ILGClasses.m

```diff
@@ -66,9 +66,20 @@ + (NSSet *)subclassesOfClass:(Class)superclass
 
 + (NSSet *)classesConformingToProtocol:(Protocol *)protocol
 {
-    return [self classesPassingTest:^BOOL(Class class) {
+    NSSet *superclassesConforming = [self classesPassingTest:^BOOL(Class class) {
         return class_conformsToProtocol(class, protocol);
     }];
+    
+    NSMutableSet *subclasses = [NSMutableSet set];
+    
+    for (Class superclass in superclassesConforming) {
+        NSSet *subclassesOfSuper = [self subclassesOfClass:superclass];
+        [subclasses addObjectsFromArray:[subclassesOfSuper allObjects]];
+    }
+    
+    NSSet *allConforming = [superclassesConforming setByAddingObjectsFromSet:subclasses];
+    
+    return allConforming;
```

Comment:
My big hesitation with this is that, where n is the total number of classes, which is probably not a small number:
- `classesPassingTest:` is O(n)
- so `subclassesOfClass:` is O(n)
- so this method is now O(n<sup>2</sup>).

If this logic is necessary, and it sounds likely, then I have some ideas for how to get it back to O(n), but I need to think about it a bit more.